### PR TITLE
Feature/ogc 1492 newsletter v2

### PR DESCRIPTION
**Pull request by Tschuppi81**
Newsletter: Show full news in email newsletter and add option to newsletter to show full content instead of tiles only

TYPE: Feature
LINK: ogc-1492

## Checklist

- [x] I have performed a self-review of my code
- [x] I considered adding a reviewer
- [x] I have updated the PO files

### CODE REVIEW

**Comment by Tschuppi81** [comment]
@Daverball I didn't change the files that fail in the linting test with 'Y065 Leave parameter "attr" unannotated rather than using "Incomplete"'. How should I proceed?
image

**Comment by Daverball** [comment]
> @Daverball I didn't change the files that fail in the linting test with 'Y065 Leave parameter "attr" unannotated rather than using "Incomplete"'. How should I proceed?

This is a new `flake8-pyi` rule which conflicts with strict mypy configs and how I write stubs, so I would disable this rule in `setup.cfg`, you can add it to the `per_file_ignores` for `*.pyi`:
```ini
per_file_ignores =
    *.pyi: B,C,MS,TC,E301,E302,E305,E501,E701,F401,F403,F405,F822,Y065
```

File: src/onegov/org/forms/newsletter.py [hunk 1]
```diff
@@ -109,10 +116,14 @@ def update_model(
...
                 super().update_model(model, request)
                 model.content['news'] = self.news.data
+                model.content['show_news_as_tiles'] = (
+                    self.show_news_as_tiles.data)
 
             def apply_model(self, model: 'Newsletter') -> None:
                 super().apply_model(model)
                 self.news.data = model.content.get('news')
+                self.show_news_as_tiles.data = bool(model.content.get(
+                    'show_news_as_tiles'))
```
Comment (Daverball) [inline src/onegov/org/forms/newsletter.py:126]:
```suggestion
                self.show_news_as_tiles.data = model.content.get(
                    'show_news_as_tiles', False)
```

File: src/onegov/town6/templates/mail_layout.pt [hunk 2]
```diff
@@ -190,9 +193,10 @@
     }
 
     hr {
-        border: 1px solid #ddd;
-        border-width: 1px 0 0 0;
+        border: 2px solid <tal:b replace="primary_color" />;
```
Comment (Daverball) [inline src/onegov/town6/templates/mail_layout.pt:196]:
```suggestion
        border: 2px solid ${primary_color};
```

**Review by Daverball — APPROVED** [review]
Just a minor nitpick

**Pull request merged**